ci/workflows/pull-request.yml

```diff
@@ -333,6 +333,25 @@ steps:
     timeout_in_minutes: 10
     retry: *auto-retry
 
+  - label: "build-image-on-pr"
+    key: "build-image-on-pr"
+    if: build.pull_request.labels includes "ci/pr/build-image-on-pr" || build.env("CI_STEPS") =~ /(^|,)ci\/pr\/build-image-on-pr(,|$$)/
+    command: |
+      # Sanitize branch name for Docker tag
+      export IMAGE_TAG=$(echo "$BUILDKITE_BRANCH" | sed 's/[^a-zA-Z0-9._-]/-/g')
+      echo "publishing new image $IMAGE_TAG from branch $BUILDKITE_BRANCH ($BUILDKITE_COMMIT)"
+      # Trigger docker.yml pipeline
+      buildkite-agent pipeline upload ci/workflows/docker.yml
```

Comment:
This is to build a **CI** image. Is this intended behavior?

Reply:
![image](https://github.com/user-attachments/assets/ccfc3ca9-f6a0-4bf2-893b-4fa0da63e9f7)

I think it is the right workflow.

Reply:
You're right.